OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -611,8 +615,7 @@ public WVec FlyStep(WAngle facing)
 
 		public WVec FlyStep(int speed, WAngle facing)
 		{
-			var dir = new WVec(0, -1024, 0).Rotate(WRot.FromYaw(facing));
-			return speed * dir / 1024;
```

Comment:
Are you sure that this change doesn't introduce rounding issues?

Reply:
Reverted